### ISSUE TITLE
prefer-module false positive when using exports as variable

### ISSUE THREAD

**Issue by ppamorim** [sindresorhus/eslint-plugin-unicorn#1969]
When using the recommended settings, unicorn is giving false postiive to the code below when running against `unicorn/prefer-module`.

Happening on `"eslint-plugin-unicorn": "40.1.0"`. I will confirm if this is still happening on the latest version, if it doesn't I will close this ticket.

Trigger:

```js
function fez() {
  fields[name] = exports.foo('value="bar"', bar);
}
```

**Comment by sindresorhus** [comment]
How is it incorrect? `exports` is a CommonJS thing.
